mobile/src/components/update-prompt.js

```diff
@@ -106,15 +117,15 @@ class UpdatePrompt extends React.Component {
               type="primary"
               style={styles.button}
               textStyle={{ fontSize: 18, fontWeight: '900' }}
-              title={'Update'}
+              title={fbt('Update', 'UpdateScreen.updateButton')}
```

Comment:
@tomlinton so we don't need to wrap this in `String` if it is being passed as a prop to a component?

Reply:
No, it just has to be wrapped for certain things like react-native <Alert />. I think basically its fine everywhere we pass it as a prop.